docs/pywbemclicmdshelp.rst

```diff
@@ -686,23 +686,26 @@ The following defines the help output for the `pywbemcli connection add --help`
 
 ::
 
-    Usage: pywbemcli connection add [COMMAND-OPTIONS]
+    Usage: pywbemcli connection add [COMMAND-OPTIONS] NAME
 
       Add a new WBEM connection definition from specified options.
 
       Create a new WBEM connection definition in the connections file from the
       specified options. A connection definition with that name must not yet
       exist.
 
-      The current connection remains unchanged by this command.
+      The NAME argument MUST exist. It define the server uri and the unique name
+      under which this server connection information will be stored. A --server
+      or --mock-server must exist because they define the server. All other
+      properties are optional.
 
-      Examples:
+      Adding a connection does not set the new connection as the current
+      connection. Use `connection select` to set a particular stored connection
```

Comment:
On "Use `connection select` to": Change to "Use the connection select command to".